db/migrate/20230605070114_create_orders.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOrders < ActiveRecord::Migration[7.0]
+  def change
+    create_table :orders do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
why foreign_key  here and not at other places?

Reply:
fixed